glossary/contractor-definition.md

```diff
@@ -0,0 +1,8 @@
+---
+type: Resource
+---
+
+# Contractor
+
+Some roles in our organization are fulfilled by contractors.
```

Comment:
I don't think this tells us anything about what a contractor is, and isn't that what a glossary definition is?

Reply:
cc @demkodo

Reply:
This also doesn't appeaar to be used anywhere; it's only linked from the glossary index.

Reply:
I'm expecting it was relevant at some point in https://github.com/publiccodenet/about/pull/571. The commit message for this file doesn't explain why this was broken out though.

I feel inclined to recommending removing this page and the link to it for this version and bringing it back when we understand it's use.

Reply:
@demkodo is fixing this in #774.